### DISCUSSION TITLE
CSS Vaccination

### DISCUSSION THREAD

**Discussion by hackit92** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#304] [Show and tell]
Que tal compañeros, quisiera mostrarles el avance que he logrado en la práctica de esta semana, iré actualizándola en los siguientes días, les deseo mucho éxito. 

https://hackit92.github.io/CssLaunchX/

**Comment by RamSantoyo** [comment]
wow te esta quedando genial, las imagenes de dónde las sacaste?

**Reply by hackit92** [reply]
Muchas gracias! Mayormente pexels y pixabay

**Comment by martinGM05** [comment]
En el navbar te recomiendo poner la imagen en absolute y un x-index=-1 y el navbar con opacity=0.9 para que se vea un efecto que el navbar es transparente. Así como la imagen de que se compartio.

**Reply by hackit92** [reply]
Fijate que el color de la barra si lo puse con transparencia pero por alguna razón no está respetándome la regla, voy a afinar todo al final pero es una excelente recomendación, gracias!

**Comment by DiegoDiem** [comment]
Una preguntota, yo estoy apenas viendo como subirlo a github pages, pero al momento de actualizar no se actualiza la página y ya se ve reflejado los cambios en gh-pages. ¿Sabes cómo solucionar eso? Porfavor.

PD: Está increíble tu página web, muy padre y buen diseño.

**Reply by hackit92** [reply]
Yo pensaría que es un tema de caché, intenta verlo en un segundo navegador o en una ventana de incógnito, no tengo mucha experiencia con github pages (suelo hacer despliegue en servidor) por lo que no se si pueda ser algo más pero descartaría primero este tema. Muchas gracias por tu comentario sobre mi página :D

**Reply by xXChAvE2Xx** [reply]
Para borrar cache puedes mantener presionada la tecla shift y actualizar la pagina, así borraras cache.. también puedes usar shift + F5.

**Reply by DiegoDiem** [reply]
@hackit92 Muchas gracias, yo creo si debe ser eso.

@xXChAvE2Xx con shift hace que borre el cache de solo esa página? No me sabía esa, gracias eh!

**Reply by xXChAvE2Xx** [reply]
@DiegoDiem Si, solo se borrara cache en la pagina que actualices.

**Comment by CRIS-117** [comment]
Está muy bien pero en mi cel no se acomoda bien las imágenes :(

**Reply by hackit92** [reply]
Es que aún no hago los mediaqueries, hasta el momento solo es la versión desktop

**Comment by JuanSalvador73** [comment]
Va muy bien, felicidades. :D

**Reply by hackit92** [reply]
Muchas gracias!
